user_guide_src/source/installation/upgrade_4xx.rst

```diff
@@ -94,6 +94,9 @@ General Adjustments
 **Helpers**
 
 - Helpers are pretty much the same as before, though some have been simplified.
+- In CI4, ``redirect()`` returns ``RedirectResponse`` instead of redirecting and terminating script execution. You must return it.
```

Comment:
```suggestion
- In CI4, ``redirect()`` returns a ``RedirectResponse`` instance instead of redirecting and terminating script execution. You must return it.
```